app/helpers/zendesk_authentication_helper.rb

```diff
@@ -0,0 +1,25 @@
+module ZendeskAuthenticationHelper
```

Comment:
Seems fine to me, though. Wondering why you chose against the base class approach? These helpers are not reusable outside the controller environment anyways (since they rely on `send_data` method and `current_user`)